### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by faloi**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @msunh

### CODE REVIEW

**Comment by msunh** [comment]
@faloi Completado

File: src/barrio.wlk [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
+//    para los métodos que devuelven algo, indicar de qué tipo es lo que devuelve;
+//    para los métodos que son acciones y no devuelven nada, especificar exactamente eso 
+//    (con un comentario que diga "no devuelve nada" o similar).
+
+
+
 class Barrio {
 
-	const property elementos = []
+	const property elementos = [] //lista - const property // setter / getter
```
Comment (faloi) [inline src/barrio.wlk:11]: ¿Lista de qué?

File: src/barrio.wlk [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
+
+
 class Barrio {
 
-	const property elementos = []
+	const property elementos = [] //lista - const property // setter / getter 
 
-	method agregarElemento(elemento) {
+	//metodo que hace - no devuelve nada, agrega elementos a la lista	
+	method agregarElemento(elemento) {  // tipo = objeto
```
Comment (faloi) [inline src/barrio.wlk:14]: Objeto es demasiado genérico, sería algo más específico.

File: src/elementos.wlk [hunk 3]
```diff
@@ -1,52 +1,68 @@
...
 
-	method fueAtacado(plaga) {
-		capacidadProduccion -= plaga.nivelDanio() * 0.1
-		if (plaga.transmiteEnfermedad()) {
-			capacidadProduccion -= 10
+	//metodo que hace | no devuelve nada
+	method fueAtacado(plaga) {							 //recibe por parámetro un Objeto tipo = Plaga | cucaracha | mosquito | garrapatas | etc
+		capacidadProduccion -= plaga.nivelDanio() * 0.1  // resta a la capacidad de produccion el nivel de danio de la plaga 
+		if (plaga.transmiteEnfermedad()) {               // y si la plaga transmite enfermedad = booleano
+			capacidadProduccion -= 10                    // les resta 10 unidades más tipo = entero
```
Comment (faloi) [inline src/elementos.wlk:56]: No se pedía explicar qué hace el método.

**Pull request closed without merging**